### ISSUE TITLE
[T&I] Loss Functions

### ISSUE THREAD

**Issue by ananya-singhh** [PurdueDualityLab/tf-maskformer#12]
### Sub-tasks
- [x] Find and understand the implementation of loss functions in the paper + codebase (including other papers / repos to look into)
- [ ] Find and verify the corresponding pre-existing loss functions in TensorFlow codebase
   - [x] Focal loss
   - [ ] Dice loss
- [ ] Implement the loss functions
  - [x] Focal loss
  - [ ] Dice loss
  - [x] Classification loss
  - [x] Mask loss
- [ ] Implement helper class 
- [ ] Do unit test for all the loss functions - generate a random tensor and use the same tensor as input for pytorch and TensorFlow variants. Make the loss values are same. please post a snippet of your output after verification.
   - [ ] Focal loss
  - [ ] Dice loss
  - [ ] Classification loss
  - [ ] Mask loss
